community/content.md

```diff
@@ -3,9 +3,10 @@
 These sections describe ways that you can create content for your JupyterHub and share them with your users.
 
 (content:nbgitpuller)=
+
 ## Distribute content with nbgitpuller
 
-You'll often want to distribute *content* (such as notebooks, scripts, sample
+You'll often want to distribute _content_ (such as notebooks, scripts, sample
```

Comment:
These are LSP-linter changes. If we're pedantic about that, I can `git commit -p` to fix this up.